background.ts

```diff
@@ -1,4 +1,5 @@
 import fs from 'fs';
+import fsPromises from 'fs/promises';
```

Comment:
This is actually accessible via `fs.promises.*` as well.

Reply:
Ok, I changed it to `fs.promises`